### ISSUE TITLE
Un'goro Soil

### ISSUE THREAD

**Issue by Kihessa** [scotty0100/wowascension-issues#415]
Upon completing the 1st quest in the chain "Assisting Arch Druid Runetotem" and clicking complete, the quest returns an error message of not meeting eligibility requirements to continue the quest "Un'goro Soil." Unsure of why this error message is returning, since this is a general quest chain.

**Comment by scotty0100** [comment]
Can't seem to replicate this.

So you are are unable to accept the quest "Un'goro Soil"?

**Comment by Kihessa** [comment]
Correct.  Female troll priest:  Upon completion of the referral quest, the quest was offered.  However, an error appeared after accepting with something as, "You don't meet the requirement for the quest".  The quest was not available thereafter.

I was able to find a reference to the error received online when I received the error:

http://valkyrie-wow.ru/boards/index.php?showtopic=3950

Date: Wed, 1 Oct 2014 14:44:54 -0700

**Comment by kamicolo** [comment]
Approved as bugged. Uncertain when or where the problem occurs. Very common is that priests get this error. Not sure if any other "Class, Faction or Race" also encounter this issue. It could be a similar problem regarding the "Bootybay Arena Trinket". Once accepted or turned in once, youre unable to redo it.
